system/lib/libc/musl/arch/emscripten/syscall_arch.h

```diff
@@ -138,3 +138,4 @@ long __syscall333(int which, ...);
 long __syscall334(int which, ...);
 long __syscall340(int which, ...);
 
+#undef SYS_futimesat
```

Comment:
That can't be the only syscall we lack and impl for though, I think? :) is something special about this one? how did you notice this was needed?